utilities/margin.css

```diff
@@ -0,0 +1,238 @@
+/*m1*/
+.m-1 {
+    margin: 0.5rem;
```

Comment:
any reason to give in rem?

Reply:
Every other padding, margin is in rem so I thought of keeping m-1 in rem too.